src/guc.c

```diff
@@ -55,6 +55,13 @@ static const struct config_enum_entry remote_data_fetchers[] = {
 	{ NULL, 0, false }
 };
 
+static const struct config_enum_entry hypertable_distributed_types[] = {
+	{ "auto", HYPERTABLE_DIST_AUTO, false },
```

Comment:
What does "auto" do? And how is it different from NULL?

Reply:
Is there a NULL for guc value? I am not aware of it

Reply:
I think this can be possible only for session variables

Reply:
Sorry, I wasn't clear. I am wondering what "auto" does. Is it similar to what we'd expect NULL (unset) to be?

Reply:
Yes, this is current behaviour. We can name it to something else